lib/extras/enc/apng.cc

```diff
@@ -195,6 +196,86 @@ void MaybeAddCICP(JxlColorEncoding c_enc, png_structp png_ptr,
   png_set_unknown_chunks(png_ptr, info_ptr, &cicp_chunk, 1);
 }
 
+bool MaybeAddSRGB(JxlColorEncoding c_enc, png_structp png_ptr,
```

Comment:
JxlColorEncoding is a struct, would we prefer to pass a pointer to it?

Reply:
That’s a good point, thank you.